pkg/policy/selectorcache.go

```diff
@@ -533,45 +673,3 @@ func (sc *SelectorCache) RemoveIdentitiesFQDNSelectors(fqdnSels []api.FQDNSelect
 		sc.updateFQDNSelector(fqdnSels[i], noIdentities)
 	}
 }
-
-// Export global functions to interface with the global selector cache
-
-// AddIdentitySelector adds the given api.EndpointSelector in to the
-// selector cache. If an identical EndpointSelector has already been
-// cached, the corresponding CachedSelector is returned, otherwise one
-// is created and added to the cache.
-func AddIdentitySelector(user CachedSelectionUser, selector api.EndpointSelector) (cachedSelector CachedSelector, added bool) {
```

Comment:
Is it possible to shift these bits that remove the singleton to a separate patch?

Reply:
Maybe, will consider tomorrow.